pkg/scheduler/internal/cache/cache.go

```diff
@@ -716,3 +718,10 @@ func (cache *schedulerCache) GetCSINodeInfo(nodeName string) (*storagev1beta1.CS
 
 	return n, nil
 }
+
+// NumNodes returns the number of nodes.
+func (cache *schedulerCache) NumNodes() int {
```

Comment:
It seems that this function is not called anywhere.

Reply:
done.